src/main/java/edu/ucla/library/iiif/auth/services/DatabaseServiceImpl.java

```diff
@@ -220,4 +275,55 @@ private RedisOptions getDbCacheClientOpts(final JsonObject aConfig) {
     private static boolean hasSingleRow(final RowSet<Row> aRowSet) {
         return aRowSet.rowCount() == 1;
     }
+
+    /**
+     * Representation of a record in the items table.
+     */
+    private static final class Item {
+
+        /**
+         * Member variable corresponding to the `uid` column of the items table.
+         */
+        private final String myUID;
+
+        /**
+         * Member variable corresponding to the `access_mode` column of the items table.
+         */
+        private final int myAccessMode;
+
+        /**
+         * Creates an item.
+         *
+         * @param aUID The item identifier
+         * @param anAccessMode The access mode to use for the item
+         */
+        @JsonCreator
+        private Item(@JsonProperty(value = RequestJsonKeys.UID, required = true) final String aUID,
```

Comment:
Curious, since I usually see these above the method (and it's easier to read)... do these have to be in the method here? It looks like maybe since they don't specify the variable name, just value. I'm not as familiar with this syntax, so asking.

Reply:
I'm not a fan of the syntax, but whenever I've tried to move them elsewhere I get a syntax error